src/main/java/roboy/dialog/states/expoStates/PassiveGreetingsState.java

```diff
@@ -0,0 +1,68 @@
+package roboy.dialog.states.expoStates;
```

Comment:
looks like copy-paste from src/main/java/roboy/dialog/states/**ordinaryStates**/PassiveGreetingsState.java
please merge both states, we don't want to have duplicate code

Reply:
Removed